src/txmempool.h

```diff
@@ -94,7 +89,17 @@ class CTxMemPoolEntry
     uint64_t nSizeWithAncestors;
     CAmount nModFeesWithAncestors;
     int64_t nSigOpCostWithAncestors;
+    size_t nTxWeight;          //!< ... and avoid recomputing tx weight (also used for GetTxSize())
+    size_t nModSize;           //!< ... and modified size for priority
+    size_t nUsageSize;         //!< ... and total memory usage
+public:
+    mutable size_t vTxHashesIdx; //!< Index in mempool's vTxHashes
+private:
+    unsigned int entryHeight;  //!< Chain height when entering the mempool
 
+    // Bool flags
+    bool hadNoDependencies;    //!< Not dependent on any other txs when it entered the mempool
```

Comment:
In commit "Re-pack the CTxMempoolEntry to be closer to optimally packed":

Also unused.